src/CreeDictionary/CreeDictionary/paradigm/manager.py

```diff
@@ -130,6 +130,59 @@ def _inflect(self, layout: ParadigmLayout, lemma: str) -> Paradigm:
         return layout.fill(template2forms)
 
 
+class ParadigmManagerWithExplicitSizes(ParadigmManager):
```

Comment:
Note: this class could be a [decorator](https://python-patterns.guide/gang-of-four/decorator-pattern/) (gang of four design pattern), but that would involve an Extract Interface refactoring to make Python's static type system happy. I decided that's just too much, after I've already done _too much_ in this PR!